File: python/q_1.py
def bmi (w,h):
    new_h = h/100
    f_bmi =  w /new_h**2
    
    bmi = round(f_bmi,2)
    if bmi <= 18:
        return bmi, "Underweight"
    elif bmi <= 25:
        return bmi, "Normal"
    elif bmi <= 30:
        return bmi, "Overweight"
    else:
        return  bmi, "Obese"
    
if __name__ == "__main__":
    weight = float(input("Please enter your weight in kg: "))
    height = float(input("Please enter your height in cm: "))
    res = bmi(weight , height)
    print (res)
    

File: c++/q_1.cpp
#include <iostream>
#include <string.h>
using namespace std;

std::string bmi(double w,double h){
    double H = h/100;
    double bmi = w/H/H;
    if(bmi<=18){
        return bmi, "Underweight";
    }
    else if(bmi <= 25){
        return bmi, "Normal";
    }
    else if(bmi <= 30){
        return bmi, "Overweight";
    }
    else{
        return bmi, "Obese";
    }
    throw std::logic_error("Method or operation is not implemented");
}
int main(){
    double weight;
    double height;
    std::cout<<"Please enter your weight in kg: ";
    std::cin >>  weight;
    std::cout<<"Please enter your height: ";
    std::cin>>height;
    string BMI = bmi(weight,height);
    std::cout<<"The result is: "<<BMI;
}
